src/tests/packagers/npm.test.ts

```diff
@@ -1003,4 +1003,123 @@ describe('NPM Packager', () => {
     expect(v6dependencies).toStrictEqual(expectedResult);
     expect(v7dependencies).toStrictEqual(expectedResult);
   });
+
+  test.each([
+    { a: 1, b: 1, expected: 2 },
+    { a: 1, b: 2, expected: 3 },
+    { a: 2, b: 1, expected: 3 },
+  ])('.add($a, $b)', ({ a, b, expected }) => {
+    expect(a + b).toBe(expected);
+  });
+
+  it.each([
+    'npm ERR! code ELSPROBLEMS',
+    'npm ERR! extraneous: foo@1.2.3 ./bar/node_modules/foo',
+    'npm ERR! missing: foo-1@1.0.0, required by foo-2@1.0.0',
+    'npm ERR! peer dep missing: foo@1.2.3',
+  ])('should ignore npm error "%s"', async (ignoredNpmError) => {
+    const v7depsList: NpmV7Deps = {
```

Comment:
feels like these long and verbose dependency lists should be fixtures

Reply:
Perhaps, I'm happy to take PRs 😛. ⌚ is scarce mostly aha